website/static/js/comment.js

```diff
@@ -561,6 +561,8 @@ var CommentListModel = function(options) {
     self.nodeId = ko.observable(options.nodeId);
     self.nodeApiUrl = options.nodeApiUrl;
 
+    self.toggle = options.toggle;
```

Comment:
The `CommentModel` has a `self.toggle` function for toggling comment replies open/closed. Even though this is the `CommentListModel` can you rename this to something like `self.togglePane` to make it more clear?